### PR TITLE
Fixes Cart Issue

### PR DESCRIPTION
**Pull request by mmohkamkarebsco**
This update fixes an issue where carts were not functioning. This was due to an issue with the EDSAutoComp() function. In the short term, setting 'autocomplete' to 'no' in the configuration settings for the plugin will allow carts to function properly.

Autocomplete is being examined for further work in a future update.

### CODE REVIEW

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 1]
```diff
@@ -1236,7 +1275,7 @@ function EDSAutoComp() {
 			console.log("Running creds false...");
 		}
 		jQuery.getJSON('[% PLUGIN_HTTP_PATH %]/opac/eds-ac.pl?type=auth', function (data) {
-			eds_sessionStorage.set('autoComp', data);
+			eds_sessionStorage.set('autoComtranslControl1p', data);
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1278]: A quick search of the code shows that this variable doesn't seem to be called anywhere else.  Is this variable causing the autocomplete issues?  It looks like as written, autoComp, which is used elsewhere, never gets set.

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 2]
```diff
@@ -1248,9 +1287,9 @@ function EDSAutoComp() {
 			console.log("Running creds true...");
 			console.log("Running autocomplete...");
 		}
-
 		jQuery('#translControl1').autocomplete({
 			source: function (request, response) {
+					console.log("In function within .autocomplete()...");
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1292]: wrap in config, or delete?

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 3]
```diff
@@ -1262,9 +1301,11 @@ function EDSAutoComp() {
 						}])
 					}
 				});
-
+					console.log("Data in promise: " + promise);
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1304]: wrap in config, or delete?

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 4]
```diff
@@ -1262,9 +1301,11 @@ function EDSAutoComp() {
...
 				promise.done(function (data) {
+						console.log("promise completed...");
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1306]: wrap in config, or delete?

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 5]
```diff
@@ -1262,9 +1301,11 @@ function EDSAutoComp() {
...
 					if (data.error) {
+							console.log("data.error : " + data.error);
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1308]: wrap in config, or delete?

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 6]
```diff
@@ -1273,6 +1314,7 @@ function EDSAutoComp() {
 						var terms = data.terms.map(function (wrapper) {
 							return wrapper.term;
 						});
+							console.log("no data.error, terms : " + terms);
```
Comment (mmabrahamson) [inline Koha/Plugin/EDS/js/EDSScript.tt:1317]: wrap in config, or delete?

**Review by mmabrahamson — COMMENTED** [review]
Found some console logs that need cleaning up, and potentially found where the autocomplete issue may be arising from.  Let me know if you've got any questions!

File: Koha/Plugin/EDS/js/EDSScript.tt [hunk 7]
```diff
@@ -455,8 +481,10 @@ function SetEDS(showInfo) {
 	jQuery('#transl1').removeClass('placeholder');
 	//advSearch
 	if (document.URL.indexOf("[% PLUGIN_HTTP_PATH %]/opac/eds-search.pl") != -1 && QueryString('q') == "") {
+		console.log('koha switchtext');
```
[the remark this answers is not in the document]
Comment (mmohkamkarebsco) [reply Koha/Plugin/EDS/js/EDSScript.tt]: Commented out

File: Koha/Plugin/EDS/js/EDSScript.tt (hunk 1 above)
Comment (mmohkamkarebsco) [reply Koha/Plugin/EDS/js/EDSScript.tt]: Changed 'autoComtranslControl1p' to 'autoComp', still dealing with same cart issues.

**Review by mmabrahamson — APPROVED** [review]
Looks good!  We do need to plan work to fix the autocomplete feature, and have a note that Autocomplete is presently not working.

**Pull request merged**